packages/primitives/text/src/Root.web.tsx

```diff
@@ -76,6 +77,12 @@ export const Text = component(
       style.textDecoration = 'underline'
     }
 
+    if (textAlign === 'right') {
+      style.textAlign = 'right'
+      style.display = 'block'
+      style.width = '100%'
+    }
+
```

Comment:
I'm afraid that we can't do it like that – it's up to a parent to align children, Text itself can't automagically become a block element and align itself

Reply:
So... a new, separate, text primitive that is block (like `<p>`) and not inline (like `<span>`)?

Reply:
Because, if I remove textAlignment inside Text, I get something like this:
<img width="462" alt="image" src="https://user-images.githubusercontent.com/13185981/62463617-c0d40100-b78a-11e9-9ce0-8dd4995410af.png">

Instead of this:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/13185981/62463570-a4d05f80-b78a-11e9-934a-90c0a09ab6af.png">
And I don't have different idea how to align it to right, because text component is stretched to full width here.

Reply:
Ok, I found soultion that doesn't need `width: 100%` nor `display: block` just `text-align: right`. I just needed to put fifth nested flexbox into TableCell :tired-pepe:
Is it ok, to just leave the part with `text-align: right` inside `Text`?

Reply:
what about some local wrapper on top of a text that has width and align? 🤔  ideally `Text` primitive should know nothing about this. something like you mentioned but not as a new primitive but rather a local/project Text wrapper.

Reply:
OK 👍 The wrapper solved the problem with aligning text! Thanks!
I'll just revert the commit with changes in Text...